services/web/server/src/simcore_service_webserver/config/server-defaults.yaml

```diff
@@ -57,3 +57,8 @@ projects:
   location: http://localhost:8043/api/specs/webserver/v0/components/schemas/project-v0.0.1.json
 session:
   secret_key: "TODO: Replace with a key of at least length 32"
+activity:
+  enabled: True
+  prometheus_host: http://prometheus
```

Comment:
I would expect ``prometheus_host: prometheus`` , i.e. without scheme.
Using [yarl](https://yarl.readthedocs.io/en/latest/api.html)  definitions
```
[scheme:]//[user[:password]@]host[:port][/path][?query][#fragment]
```
In any case, all internal comms  between restAPIs are http

Reply:
let's put the scheme there, as it has to be defined somewhere, and yarl likes it together with the host. otherwise it has to be defined somewhere in a separate config entry and makes it a bit cumbersome... or what do you think?